app/seasonal-planting-guide/page.tsx

```diff
@@ -1,8 +1,17 @@
 'use client';
 
 import React, { useState } from 'react';
-import FilterDropdown from '@/components/FilterDropdown';
+import FilterDropdownMultiple from '@/components/FilterDropdownMultiple';
+import FilterDropdownSingle from '@/components/FilterDropdownSingle';
 import { PlantList } from '@/components/PlantList';
+import SearchBar from '@/components/SearchBar/SearchBar';
+import { DropdownOption } from '@/types/schema';
+import {
+  FilterContainer,
+  HeaderContainer,
+  PageContainer,
+  StateOptionsContainer,
+} from './styles';
 
 const SeasonalPlantingGuide = () => {
```

Comment:
to be consistent with the rest of the codebase, this should be 

`export default const SeasonalPlantingGuide`